### PR TITLE
Add MIDI.js dependency. Start tidying up the accessible music demo.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

**Comment by seanlip** [comment]
@NeilFraser -- could you please check just the Makefile part of this, since it's the only thing that would affect other devs? Thanks!

File: appengine/accessible/js/blocks.js [hunk 1]
```diff
@@ -19,23 +19,20 @@
...
+goog.require('Blockly');
+goog.require('Blockly.JavaScript');
 
 var MUSIC_DUMMY_TOOLTIP = 'Dummy tooltip';
 var MUSIC_DUMMY_HELPURL = 'Dummy help URL';
 
-/**
- * Common HSV hue for all blocks in this category.
- */
-Blockly.Blocks.music.HUE = 20;
```
Comment (NeilFraser) [inline appengine/accessible/js/blocks.js]: Why the move to repeated constants?

**Comment by NeilFraser** [comment]
LGTM

File: appengine/accessible/js/blocks.js (hunk 1 above)
Comment (seanlip) [inline appengine/accessible/js/blocks.js]: I noticed that, in the other games (turtle/maze etc.), the individual block colors were hardcoded per-block, so I thought it best to be consistent. What would you prefer? I'm fine either way.

Comment (NeilFraser) [inline appengine/accessible/js/blocks.js]: Originally all blocks were hardcoded.  Blockly eventually got upgraded.  Looks like Blockly Games never got upgraded.  Thanks for pointing that out, I'll fix than now.

Comment (seanlip) [inline appengine/accessible/js/blocks.js]: OK. I'll push this as-is, then, so that the fix can be done consistently. Thanks!

Comment (NeilFraser) [inline appengine/accessible/js/blocks.js]: Good call.  Done.

**Pull request merged**